src/ZipkinOpenTracing/Tracer.php

```diff
@@ -97,15 +153,15 @@ public function inject(OTSpanContext $spanContext, $format, &$carrier)
             return $injector($spanContext->getContext(), $carrier);
         }
 
-        throw new InvalidArgumentException(sprintf(
+        throw new \InvalidArgumentException(sprintf(
```

Comment:
Any reason for this change?

Reply:
Do you prefer having?

```
use InvalidArgumentException;

throw new InvalidArgumentException();
```

Instead?

Reply:
Personally yes but my concern was more about what was the reason behind doing it in this PR. Not strong opinion about it tho.